app/assets/stylesheets/modules/panel_list.css.scss

```diff
@@ -1,4 +1,4 @@
-$panel-list-item-height: 85px;
+$panel-list-item-height: 75px;
```

Comment:
That's on the collection. I was looking at the granule thumbnail. Both of them probably need to be fixed.